### ISSUE TITLE
pyyaml and pip not for Python 2.6 - Python 2 End Of Life (EOL) 

### ISSUE THREAD

**Issue by ghost** [openaire/Generic-Matomo-Tracker#4]
In the README it says "_Requires python 2.6 or 2.7._" Unfortunately pyyaml is only available for Python 2.7 and Python 3, not for Python 2.6. Same goes for pip. Pip is avaialbale since Python 2.7.9.

Actually, Python 2 reached [End Of Life on 1 Janary 2020](https://www.python.org/doc/sunset-python-2/) and is no longer supported or maintained. It is really time to move this to Python 3.

**Comment by dimitrispie** [comment]
> In the README it says "_Requires python 2.6 or 2.7._" Unfortunately pyyaml is only available for Python 2.7 and Python 3, not for Python 2.6. Same goes for pip. Pip is avaialbale since Python 2.7.9.
> 
> Actually, Python 2 reached [End Of Life on 1 Janary 2020](https://www.python.org/doc/sunset-python-2/) and is no longer supported or maintained. It is really time to move this to Python 3.

Thanks @ajbosman for the comment. We will plan to update to Python 3 asap.

**Comment by d-schiffner** [comment]
@dimitrispie We have implemented a python3 version of it with some major restructuring though. Its currently hosted at our gitlab server.

Great! Can you provide the link?

**Comment by fjorba** [comment]
Any news about the python 3 version, please?  Where is it hosted?  I've googled for openaire, gitlab and matomo with no success.

Thanks!

**Comment by d-schiffner** [comment]
@fjorba Sorry, have not been working on that project lately. I will look into the sources and try to create a repository and link it here.

**Comment by dimitrispie** [comment]
> @fjorba Sorry, have not been working on that project lately. I will look into the sources and try to create a repository and link it here.

The new version of the tracker requires python 3.

**Comment by d-schiffner** [comment]
@dimitrispie Yes, i see. In our version we also changed the processing of the hits, as we ran into some issues with the workers. Maybe it is still of interest, though:

https://github.com/DIPF-educs/openaire-matomo-tracker
